app/gen-server/lib/homedb/UsersManager.ts

```diff
@@ -279,6 +279,10 @@ export class UsersManager {
         user.name = props.name;
         needsSave = true;
       }
+      if (Object.keys(props.options ?? []).length > 0) {
```

Comment:
```suggestion
      if (Object.keys(props.options ?? {}).length > 0) {
```

Because you want to list keys of an object.

Otherwise:
```suggestion
      if (props.options && Object.keys(props.options).length > 0) {
```

Reply:
You are correct ! will remove this